crates/bevy_window/src/monitor.rs

```diff
@@ -0,0 +1,63 @@
+use bevy_ecs::component::Component;
+use bevy_ecs::prelude::ReflectComponent;
+use bevy_math::{IVec2, UVec2};
+use bevy_reflect::Reflect;
+
+#[cfg(feature = "serialize")]
+use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
+
+/// Represents a monitor attached to the system, which can be used to create windows.
+///
+/// This component is synchronized with `winit` through `bevy_winit`, but is effectively
+/// read-only as `winit` does not support changing monitor properties.
+#[derive(Component, Debug, Clone, Reflect)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+#[reflect(Component)]
+pub struct Monitor {
+    /// The name of the monitor
+    pub name: Option<String>,
+    /// The height of the monitor in physical pixels
+    pub physical_height: u32,
+    /// The width of the monitor in physical pixels
+    pub physical_width: u32,
+    /// The position of the monitor in physical pixels
+    pub physical_position: IVec2,
+    /// The refresh rate of the monitor in millihertz
+    pub refresh_rate_millihertz: Option<u32>,
+    /// The scale factor of the monitor
+    pub scale_factor: f64,
+    /// The video modes that the monitor supports
+    pub video_modes: Vec<VideoMode>,
+}
+
+/// A marker component for the primary monitor
+#[derive(Component, Debug, Clone, Reflect)]
+#[reflect(Component)]
+pub struct PrimaryMonitor;
```

Comment:
If you're going with it above, also add `#[doc(alias = "WinitPrimaryScreen")]`/`#[doc(alias = "WinitPrimaryDisplay")]` here :)